docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -106,6 +113,15 @@ DELETE FROM                          all                delete
 UPDATE                               all                update
 ==================================== ========== ======= ==================== ===================================================
 
+Permissions required for executing functions and PTFs (Polymorphic Table Function):
```

Comment:
If you're adding a new table, please make it a list-table. You can search in the docs for `list-table` for examples and formatting.

Reply:
I used new `list-table`, please confirm if I did it right